class.jetpack.php

```diff
@@ -4692,9 +4701,13 @@ function verify_xml_rpc_signature() {
 			$body = http_build_query( stripslashes_deep( $post_data ) );
 		} elseif ( is_null( $this->HTTP_RAW_POST_DATA ) ) {
 			$body = file_get_contents( 'php://input' );
+			error_log( print_r( array(
```

Comment:
This feels like an awkward way to log errors -- wouldn't it be simpler to concatenate the `print_r( $body, true )` onto the string, rather than making a bigger array?